install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# By default, this script will install the latest version of run to $HOME/.local/bin.
+install_dir="$HOME/.local/bin"
+version="latest"
+
+# Get the actual version from the latest release
+if [ "$version" = "latest" ]; then
+    version=$(curl -s https://api.github.com/repos/amonks/run/releases/latest | grep tag_name | cut -d '"' -f 4)
+fi
+
+# Detect OS and Architecture
+os=""
+arch=$(uname -m)
+os_type=$(uname -s)
+
+case "$os_type" in
+    Linux*)  os="Linux";;
+    Darwin*) os="Darwin";;
+    *) echo "Unsupported OS" && exit 1;;
```

Comment:
as a freebsd user, I'd like to add freebsd support before merging

(happy to do this myself: not saying "pls eric go find a freebsd system to test on")

Reply:
ah; I see that we don't actually build for freebsd 😅
I'll make a separate ticket for that 👍